src/components/User/Fees/components/CollectedFeesWithdraw.vue

```diff
@@ -256,9 +265,9 @@ export default {
     maxWithdrawalAmountHint () {
       const formatted = formatAssetAmount(
         this.selectedBalanceAttrs.available,
-        this.selectedBalanceAttrs.assetCode,
+        this.selectedBalanceAttrs.assetCode
       )
-      return `Max amount is ${formatted}`
+      return globalize('collected-fees-withdraw.max-amount')`${formatted}`
```

Comment:
use interpolation
Are you sure it works?

Reply:
Are you sure you know js?

Reply:
done